frontend/src/components/VerticalWord/CrosswordBoxContainer.tsx

```diff
@@ -67,10 +83,11 @@ const CrosswordBoxContainer = ({ crossword }: CrosswordBoxContainerProps) => {
 
   return (
     <CrosswordContainer>
-      {grid.map( (row, i ) => 
+      {grid.map( (row : any, i : number) => 

```

Comment:
lets try not to use any for types here we can use string[]

Reply:
for sure, this is a wait why the hell is it giving me type errors _now_ move. updated